DateOnlyTimeOnly.AspNet.sln

```diff
@@ -9,6 +9,7 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 	ProjectSection(SolutionItems) = preProject
 		.editorconfig = .editorconfig
 		Directory.Build.props = Directory.Build.props
+		LICENSE = LICENSE
```

Comment:
Also revert this please. This file is not expected to ever change, so there's no point in cluttering space in Solution Explorer with it.

```suggestion
```